charts/mattermost-operator/README.md

```diff
@@ -17,57 +17,128 @@ https://kubernetes.io/docs/home/
-
+To start, copy [mattermost-helm/charts/mattermost-operator/values.yaml](https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-operator/values.yaml) and name it `config.yaml`. This will be your configuration file for the Mattermost Operator chart. The default values will deploy the resources necessary to run the Mattermost Operator.
 
 # 3. Install
 
 After adding the Mattermost repo (see section 1.2) you can install a version of the preferred chart by running:
 
-```bash
+```console
 $ helm install <your-release-name> mattermost/mattermost-operator -n <namespace_name>
 ```
 
 For example:
-```bash
+```console
 $ helm install mattermost-operator mattermost/mattermost-operator -n mattermost-operator
 ```
 
 If no version is specified the latest version will be installed.
 
+---
 
 To run with your custom `config.yaml`, install using:
 
-```bash
+```console
 helm install <your-release-name> mattermost/mattermost-operator -f config.yaml -n mattermost-operator
 ```
 
+## 3.1 Upgrade
+
+### Upgrading the CRDs
+
+Helm does not upgrade the CRDs during a release upgrade. Before you upgrade a release, preform the following to ensure you have the appropriate CRDs available.
+
+1. Pull the chart sources:
+
+    ```console
+    helm pull mattermost/mattermost-operator --untar --version <VERSION_HERE>
+    ```
+
+2. Change your working directory to mattermost-operator:
+
+    ```console
+    cd mattermost-operator
+    ```
+
+3. Update the CRDs on the cluster:
+
+    ```console
+    kubectl apply -f crds/
+    ```
+
+### Upgrading the release
+
 To upgrade an existing release, modify the `config.yaml` with your desired changes and then use:
-```bash
+```console
 helm upgrade -f config.yaml <your-release-name> mattermost/mattermost-operator -n mattermost-operator
 ```
 
-If the `mysql-operator` and `minio-operator` are enabled their namespaces will be automatically created.
-
-## 3.1 Uninstalling Mattermost Operator Helm Chart
+## 3.2 Uninstalling Mattermost Operator Helm Chart
 
 If you are done with your deployment and want to delete it, use `helm delete <your-release-name>`. If you don't know the name of your release, use `helm ls -A` to find it across all namespaces. Make sure to pass the namespace in the helm delete command.
 
-
 # 4. Developing
 
 If you are going to modify the helm charts, it is helpful to use `--dry-run` (doesn't do an actual deployment) and `--debug` (print the generated config files) when running `helm install`.
 
 Helm has partial support for pulling values out of a subchart via the requirements.yaml. It also has limited support for pushing values into subcharts. It does not support using templating inside a values.yaml file.
 
 We recommend using [kind](https://github.com/kubernetes-sigs/kind) for local development if you do not have access to Kubernetes cluster running in the cloud.
+
+## Chart v1.0.0 Upgrade Notes
```

Comment:
Tagging @cwarnermm and @coltoneshaw for visibility. Carrie, I will be working with you to update our docs once this is merged.